servant-client/test/ghcjs/build-test-server.sh

```diff
@@ -0,0 +1,10 @@
+#/usr/bin/env bash
+
+set -o errexit
+
+cd test/ghcjs
```

Comment:
Could this also be folded into the `.travis.sh` file (i.e. have the test server be built *before* the tests are run)?